docs/SUMMARY.md

```diff
@@ -1,15 +1,17 @@
 # Table of Contents
 
-* [CSI repository welcome](book_files/csi_block_storage_kc_welcome.md)
+* [Welcome](book_files/csi_block_storage_kc_welcome.md)
```

Comment:
`csi_block_storage_kc_welcome.md` still contains `1.6.0`, instead of `1.7.0`

Reply:
DONE